.github/workflows/build.yml

```
@@ -0,0 +1,40 @@
+on:
+  - push
+  - pull_request
+
+name: Build
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ['2.7', '3.1', 'jruby', 'jruby-head']
```

```suggestion
        ruby: ['2.7', '3.1', 'jruby-9.1', 'jruby-9.2', 'jruby-9.3', 'jruby-head']
```

Suggest this because JRuby 9.1.17.0 is known to pass tests, while 9.2 and later have breakage. We should allow 9.2/9.3/jruby-head to fail for now, until we can fix those tests.

---

Oops, I've been slandering JRuby unfairly. It turns out the specs fail with `nokogiri` starting at `>= 1.11.0`. If we lock to `~> 1.8, >= 1.8.3, < 1.11.0` then the specs pass with all currently released versions of JRuby.

I think there are still some problems with the Cucumber console tests with newer JRuby, will look into that later.